Lib9c/Model/Event/EventDungeonInfo.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using Bencodex.Types;
+using Libplanet;
+using Nekoyume.Action;
+using Nekoyume.Model.State;
+
+namespace Nekoyume.Model.Event
+{
+    public class EventDungeonInfo : IState
+    {
+        public static Address DeriveAddress(Address address, int dungeonId)
+        {
+            return address.Derive($"event_dungeon_info_{dungeonId}");
+        }
+
+        private int _remainingTickets;
+        private int _clearedStageId;
```

Comment:
It only have the last cleared stageId. is this order of stage id not changed? it would be better save list of stage id.

Reply:
I plan to delegate that part to using this class.